infrahub_sdk/branch.py

```diff
@@ -22,6 +30,7 @@ class BranchData(BaseModel):
     is_default: bool
     has_schema_changes: bool
     graph_version: int | None = None
+    status: BranchStatus
```

Comment:
I wonder if we should have this default to `BranchStatus.OPEN` without something like that I think it could be problematic to use the SDK with earlier versions of Infrahub?